### ISSUE TITLE
Error in the Console 1.16.1

### ISSUE THREAD

**Issue by CroaBeast** [LapisPlugins/AFKPlus#6]
This error appears so frecuently, what's that means?
```
>.... [22:43:56 WARN]: [AFKPlus] Task #532837 for AFKPlus v3.1.8 generated an exception
java.lang.NullPointerException: null
at net.lapismc.afkplus.playerdata.AFKPlusPlayer.broadcast(AFKPlusPlayer.java:225) ~[?:?]
at net.lapismc.afkplus.playerdata.AFKPlusPlayer.startAFK(AFKPlusPlayer.java:148) ~[?:?]
at org.bukkit.craftbukkit.v1_16_R1.scheduler.CraftTask.run(CraftTask.java:99) ~[patched_1.16.1.jar:git-Paper-99]
at org.bukkit.craftbukkit.v1_16_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:468) ~[patched_1.16.1.jar:git-Paper-99]
at net.minecraft.server.v1_16_R1.MinecraftServer.b(MinecraftServer.java:1291) ~[patched_1.16.1.jar:git-Paper-99]
at net.minecraft.server.v1_16_R1.DedicatedServer.b(DedicatedServer.java:377) ~[patched_1.16.1.jar:git-Paper-99]
at net.minecraft.server.v1_16_R1.MinecraftServer.a(MinecraftServer.java:1212) ~[patched_1.16.1.jar:git-Paper-99]
at net.minecraft.server.v1_16_R1.MinecraftServer.v(MinecraftServer.java:1000) ~[patched_1.16.1.jar:git-Paper-99]
at net.minecraft.server.v1_16_R1.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.1.jar:git-Paper-99]
at java.lang.Thread.run(Thread.java:748) [?:1.8.0_242]
```

**Comment by CroaBeast** [comment]
also the plugin kicks every single user when is afk:
```
>.... [17:44:42 INFO]: [AFKPlus] [AFK+] ximojcml is now AFK
>.... [17:44:43 INFO]: CONSOLE issued server command: /kick ximojcml You have been kicked as you have been AFK for 5 minutes
>.... [17:44:43 INFO]: ximojcml lost connection: You have been kicked as you have been AFK for 5 minutes
>.... [17:44:43 INFO]: ✖ Espero que ximojcml vuelva pronto.
>.... [17:44:43 INFO]: Console kickeado ximojcml por You have been kicked as you have been AFK for 5 minutes.
```

**Comment by Dart2112** [comment]
You seem to have the time to action set to a low number, could you show your permissions for that player from the config

**Comment by CroaBeast** [comment]
the plugin has permissions? I didn't see in the spigot page

**Comment by Dart2112** [comment]
yup, they are all on the config towards the bottom.
They are a bit different from other plugins though. Chuck a message in the discussion on the spigot page if you need any help with those

**Comment by CroaBeast** [comment]
so, afkplus.player is the permission for default group? In the config says afkplus,player 'with the comma'

**Comment by Dart2112** [comment]
right on
whoever you give the afkplus.player permission too will have all of the afkplus,player settings applied to them

**Comment by CroaBeast** [comment]
ok, I'll try that, and whats that spam in the console?

**Comment by Dart2112** [comment]
what spam? do you mean the getting kicked spam?

**Comment by CroaBeast** [comment]
I mean the first issue when I open the issue

> This error appears so frecuently, what's that means?
> 
> ```
> >.... [22:43:56 WARN]: [AFKPlus] Task #532837 for AFKPlus v3.1.8 generated an exception
> java.lang.NullPointerException: null
> at net.lapismc.afkplus.playerdata.AFKPlusPlayer.broadcast(AFKPlusPlayer.java:225) ~[?:?]
> at net.lapismc.afkplus.playerdata.AFKPlusPlayer.startAFK(AFKPlusPlayer.java:148) ~[?:?]
> at org.bukkit.craftbukkit.v1_16_R1.scheduler.CraftTask.run(CraftTask.java:99) ~[patched_1.16.1.jar:git-Paper-99]
> at org.bukkit.craftbukkit.v1_16_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:468) ~[patched_1.16.1.jar:git-Paper-99]
> at net.minecraft.server.v1_16_R1.MinecraftServer.b(MinecraftServer.java:1291) ~[patched_1.16.1.jar:git-Paper-99]
> at net.minecraft.server.v1_16_R1.DedicatedServer.b(DedicatedServer.java:377) ~[patched_1.16.1.jar:git-Paper-99]
> at net.minecraft.server.v1_16_R1.MinecraftServer.a(MinecraftServer.java:1212) ~[patched_1.16.1.jar:git-Paper-99]
> at net.minecraft.server.v1_16_R1.MinecraftServer.v(MinecraftServer.java:1000) ~[patched_1.16.1.jar:git-Paper-99]
> at net.minecraft.server.v1_16_R1.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.1.jar:git-Paper-99]
> at java.lang.Thread.run(Thread.java:748) [?:1.8.0_242]
> ```

**Comment by Dart2112** [comment]
ahh, that's an issue with players being marked as AFK while they aren't on the server for some reason.
I've implemented a fix and it will be in the next update

**Comment by CroaBeast** [comment]
ok, thanks btw, with the permission I think that will be fixed

**Comment by Dart2112** [comment]
Fixed in 3.1.9
Update pushed to spigot and auto updater
